docs/source/admin/index.rst

```diff
@@ -21,22 +21,29 @@ Traffic Control is distributed in source form for the developer, but also as a b
 
 When installing a complete CDN from scratch, a sample recommended order is:
 
+#. Traffic Ops DB (Postgresql)
+#. `InfluxDB [Optional] <https://github.com/influxdata/influxdb>`_
+#. Traffic Vault (Riak) [Optional]
+#. Fake Origin [Optional]
 #. Traffic Ops
-#. Traffic Vault (Riak)
 #. Traffic Portal
+#. Initial Traffic Ops Dataset Setup [if applicable]
 #. Traffic Monitor
 #. Apache Traffic Server Mid-Tier Caches
 #. Apache Traffic Server Edge-Tier Caches
+#. Grove [Optional]
 #. Traffic Router
-#. Traffic Stats
+#. `InfluxDB-relay [Optional] <https://github.com/influxdata/influxdb-relay>`_
+#. Traffic Stats [Optional]
 
 Once everything is installed, you will need to configure the servers to talk to each other. You will also need Origin server(s), from which the Mid-Tier Cache(s) will obtain content. An Origin server is simply an HTTP(S) server which serves the content you wish to cache on the CDN.
 
 .. toctree::
 	:maxdepth: 3
 	:glob:
 
-	traffic_ops.rst
+	environment_creation.rst
+	traffic_ops/*
```

Comment:
I didn't catch this earlier, but the new glob doesn't match anything and removing that top line leaves `traffic_ops.rst` orphaned from the toc tree - I think this should just be a one-line change adding `environment_creation.rst`

Reply:
Addressed with commit